src/btree/bt_curprev.c

```diff
@@ -334,14 +351,14 @@ __cursor_var_prev(WT_CURSOR_BTREE *cbt, bool newpage)
 			return (WT_NOTFOUND);
 		__cursor_set_recno(cbt, cbt->last_standard_recno);
 		cbt->cip_saved = NULL;
-		goto new_page;
+		goto retry_insert;
 	}
 
 	/* Move to the previous entry and return the item. */
 	for (;;) {
 		__cursor_set_recno(cbt, cbt->recno - 1);
 
-new_page:	if (cbt->recno < cbt->ref->ref_recno)
+retry_insert:	if (cbt->recno < cbt->ref->ref_recno)
```

Comment:
@keithbostic, Could you review this change, I wanted your attention as I have doubt.  My doubt is "retry_insert", should it be labelled on line 364 instead of line 361 ? similar to next.